Add how to add other language apart from Japanese (#30)
themoeway/local-audio-yomichan: README.md

        > **However, Opus is NOT compatible with AnkiMobile (iOS), Android 4, and AnkiWeb.**
        > If you use any of these, please use the 2nd option (MP3 audio) below.
        >

>         > -   **For people who learn Japanese, download the files from [this torrent](https://nyaa.si/view/1681655)**.

Reviewer: ```suggestion
        > -   **For Japanese audio, download the files from [this torrent](https://nyaa.si/view/1681655)**.
```

        >     ```
        >
        >     </details>
        >

>         > -   **For people who learn other language**, use the magnet link below and select to download _only_ zip file of the language under the folder Forvo_pronunciation > export > opus > [iso 639 code of language that you want to learn.zip](https://en.wikipedia.org/wiki/List_of_ISO_639_language_codes):

Reviewer: ```suggestion
        > -   **For audio of other languages**, use the magnet link below containing Forvo audio and select the zip file of the language you want under the folder `Forvo_pronunciation > export > opus > [iso 639 code of language that you want to learn.zip](https://en.wikipedia.org/wiki/List_of_ISO_639_language_codes)`:
```

        >     <details><summary>Magnet link</summary>
        >
        >     ```
        >     magnet:?xt=urn:btih:255A1B495719DD3CE0CF8AA708FE6837AD9F57E1&tr=http%3A%2F%2Fbt2.t-ru.org%2Fann%3Fmagnet&dn=%D0%9D%D0%B0%D0%B1%D0%BE%D1%80%20%D0%BF%D1%80%D0%BE%D0%B8%D0%B7%D0%BD%D0%BE%D1%88%D0%B5%D0%BD%D0%B8%D0%B9%20%D0%BE%D1%82%20Forvo.com%20%D0%B4%D0%BB%D1%8F%20GoldenDict%20%D0%BD%D0%B0%20%D0%B2%D1%81%D0%B5%D1%85%20%D1%8F%D0%B7%D1%8B%D0%BA%D0%B0%D1%85
        >     ```
        >
        >     </details>
        </details>

    1.  <details>
        <summary>MP3 audio (4.9 GiB)</summary>

        > Older and less efficient codec, but needed for compatibility with pretty much all devices.

>         > -   **For people who learn Japanese, download the files from [this torrent](https://nyaa.si/view/1681654)**.

Reviewer: ```suggestion
        > -   **For Japanese audio, download the files from [this torrent](https://nyaa.si/view/1681654)**.
```

        >     ```
        >
        >     </details>
        >

>         > -   **For people who learn other language**, use the magnet link below and select to download _only_ zip file of the language under the folder Forvo_pronunciation > export > mp3 > [iso 639 code of language that you want to learn.zip](https://en.wikipedia.org/wiki/List_of_ISO_639_language_codes):

Reviewer: ```suggestion
        > -   **For audio of other languages**, use the magnet link below containing Forvo audio and select the zip file of the language you want under the folder `Forvo_pronunciation > export > opus > [iso 639 code of language that you want to learn.zip](https://en.wikipedia.org/wiki/List_of_ISO_639_language_codes)`:
```

        >     <details><summary>Magnet link</summary>
        >
        >     ```
        >     magnet:?xt=urn:btih:255A1B495719DD3CE0CF8AA708FE6837AD9F57E1&tr=http%3A%2F%2Fbt2.t-ru.org%2Fann%3Fmagnet&dn=%D0%9D%D0%B0%D0%B1%D0%BE%D1%80%20%D0%BF%D1%80%D0%BE%D0%B8%D0%B7%D0%BD%D0%BE%D1%88%D0%B5%D0%BD%D0%B8%D0%B9%20%D0%BE%D1%82%20Forvo.com%20%D0%B4%D0%BB%D1%8F%20GoldenDict%20%D0%BD%D0%B0%20%D0%B2%D1%81%D0%B5%D1%85%20%D1%8F%D0%B7%D1%8B%D0%BA%D0%B0%D1%85
        >     ```
        >

>         >     </details>

Reviewer: Why is this repeated twice? Is the audio in the torrent mp3 or ogg?

        </details>

    If you have never downloaded from a torrent before, I highly recommend using the
    [qBittorrent](https://www.qbittorrent.org/) client.

> 2. Extract the `.tar.xz` (Japanese), or `.zip` file for other language.

Reviewer: ```suggestion
2. Extract the `.tar.xz` (Japanese), or `.zip` file for other languages.
```

        ![image](./img/view_files.png)

>     * For people who learn Japanese, copy the entire `user_files` folder (from the second step) into this folder.

Reviewer: ```suggestion
    * For Japanese, copy the entire `user_files` folder (from the second step) into this folder.
```

>         <details> <summary>Expected file structure for people who learn Japanese <i>(click here)</i></summary>

Reviewer: ```suggestion
        <details> <summary>Expected file structure for Japanese audio <i>(click here)</i></summary>
```

        </details>

>     * For people who learn other language: change the name of folder from the second step to `forvo_files` and copy the entire folder _into_ `user_files` folder.

Reviewer: ```suggestion
    * For other languages, change the name of the folder from the second step to `forvo_files` and copy the entire folder _into_ the `user_files` folder.
```

>         <details> <summary>Example file structure for people who learn other language <i>(click here)</i></summary>

Reviewer: ```suggestion
        <details> <summary>Example file structure for other languages' audio <i>(click here)</i></summary>
```
